### DISCUSSION TITLE
Is there support for specifying coordinates instead of having to manually select with the mouse?

### DISCUSSION THREAD

**Discussion by FM1337** [dynobo/normcap#484] [Q&A] [answered]
Title is self-explanatory but since there isn't an API for this package and the help output for there CLI doesn't indicate that there is a way to specify coordinates, I'm hoping there might just be something hidden that would allow it?

**Comment by dynobo** [comment]
Hi @FM1337 , thanks for your question.

Unfortunately no, that's not possible at the moment.

May I ask, what your use-case is, to better understand your requirements?

**Reply by FM1337** [reply]
So it turns out I might not actually need your library/app for this (I went with pytesseract and PIL's ImageGrab and now I just need a way to automatically find a window's coordinates and then mathematically determine each letter's position because tesseract seemingly can't figure out the grid by itself). 

Originally I was looking for a way to read text from a screen and while this could do it, it required manual interaction whereas I needed to automate it for a script to try and figure out if the developers of Wordament had either messed up a level accidently or intentionally by running through a gigantic list of all possible words including ones that weren't in the official scrabble wordlist automatically rather than manually having to do it by hand. (All because I'm at 248/250 words in this one level and the last two according to every solver I've tried (including the one I wrote) do not work)

**Reply by dynobo** [reply, marked as answer]
Wow, interesting use case, thanks for sharing!

I'm glad you found a solution for the OCR part. Glueing some screenshot tool to plain tesseract (or pytesseract) whould have been my recommendation here, too.

Regarding grid-detection, I've planned to add something similar to NormCap (table detection, see #17), but it turned out to be quite complex to do this in a robust way. 
In your case, the grid is probably with fixed dimension, then manually measure top-left and cell width/height and iterating over that might be a quick win.

Good luck! :+1:
